compiler/testData/diagnostics/tests/operatorsOverloading/assignOperatorOverloadDisabled.kt

```diff
@@ -0,0 +1,22 @@
+// FIR_IDENTICAL
```

Comment:
You should switch the feature off explicitly

Reply:
I removed `FIR_IDENTICAL` feature in [compiler/testData/diagnostics/tests/operatorsOverloading/assignOperatorOverloadDisabled.kt](https://github.com/JetBrains/kotlin/pull/4890/files#diff-20b915b704de1013df4264283f347d7a1c8e3905a5d8f10f5a6f6c8468e272d5)